README.md

```diff
@@ -50,35 +50,69 @@ credentials as well as utility methods to create them and to get Application Def
 
 ## Quickstart
 
+### Using Maven
```

Comment:
Thanks!

Thoughts on also promoting libraries-bom here (perhaps a small section)? We can mention that if you use libraries-bom, then auth-bom is imported as well. Otherwise, you should manually import the bom (like below)

Reply:
Sounds good. I'll add a section.

Reply:
Ah, I think it's not as trivial to ensure the snippet with libraries-bom has an up-to-date `<version/>` since it's not managed in versions.txt.

Other repos such as google-cloud-java use the [config yaml](https://github.com/googleapis/google-cloud-java/blob/7d99222b94664eb74edb5c592f85ce05bf9bfdff/generation_config.yaml#L3).
We also have the case of spring-cloud-gcp which seems to [rely on renovate-bot](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/renovate.json). I think it's better to wait for such PR and otherwise raise an issue to keep it up to date.

Reply:
Ah yeah makes sense. Maybe we don't need to have a code reference here. We could just reference this page: https://cloud.google.com/java/docs/bom